apps/src/gamelab/blocks.js

```diff
@@ -40,19 +44,13 @@ export default {
         init: function () {
           this.setHSV(...CATEGORIES[category].color);
           if (methodCall) {
```

Comment:
Is this redundant now that the blocks are defined as `'{THIS} move up'` below?

Reply:
It could be inferred from the presence of the string `{THIS}` in the block text, but I'd rather it remain a bit more explicit than that.

I also considered indicating method calls by adding a flag on an argument marking it as the object on which the method was being called, rather than a normal parameter. `methodCall` would then be more directly inferred from the presence of a `this` argument, and the block text string could call it `{SPRITE}` or whatever instead of always calling it `{THIS}`.

I'll see how it feels the next couple times I make a method call block.